packages/ui/src/app/pages/Proposals/PastProposals.tsx

```diff
@@ -1,34 +1,44 @@
-import React, { useRef } from 'react'
+import React, { useRef, useState } from 'react'
 
 import { AppPage } from '@/app/components/AppPage'
 import { ActivitiesBlock } from '@/common/components/Activities/ActivitiesBlock'
+import { FilterPageHeader } from '@/common/components/forms/FilterBox'
 import { Loading } from '@/common/components/Loading'
 import { ContentWithSidepanel, MainPanel } from '@/common/components/page/PageContent'
-import { PageHeader } from '@/common/components/page/PageHeader'
-import { PageTitle } from '@/common/components/page/PageTitle'
 import { SidePanel } from '@/common/components/page/SidePanel'
 import { useActivities } from '@/common/hooks/useActivities'
 import { AddProposalButton } from '@/proposals/components/AddProposalButton'
+import { ProposalEmptyFilter, ProposalFilters } from '@/proposals/components/ProposalFilters'
 import { ProposalList } from '@/proposals/components/ProposalList'
 import { usePastProposals } from '@/proposals/hooks/usePastProposals'
 
 import { ProposalsTabs } from './components/ProposalsTabs'
 
 export const PastProposals = () => {
-  const { proposals, isLoading } = usePastProposals()
+  const searchSlot = useRef<HTMLDivElement>(null)
+  const [filters, setFilters] = useState(ProposalEmptyFilter)
+
+  const { proposals, types, stages, proposers, isLoading } = usePastProposals({ filters })
   const activities = useActivities()
+
   const sideNeighborRef = useRef<HTMLDivElement>(null)
 
   return (
     <AppPage>
-      <PageHeader>
-        <PageTitle>Proposals</PageTitle>
+      <FilterPageHeader ref={searchSlot} title="Proposals">
         <AddProposalButton />
         <ProposalsTabs />
-      </PageHeader>
+      </FilterPageHeader>
```

Comment:
I've just noticed that the search box is bigger than the "add new proposal" button.